src/xpring-client.ts

```diff
@@ -17,12 +16,11 @@ class XpringClient {
    * The XpringClient will use gRPC to communicate with the given endpoint.
    *
    * @param grpcURL The URL of the gRPC instance to connect to.
-   * @param useNewProtocolBuffers If `true`, then the new protocol buffer implementation from rippled will be used. Defaults to false.
    */
-  public constructor(grpcURL: string, useNewProtocolBuffers = false) {
```

Comment:
What happened to `useNewProtocolBuffers`?

Reply:
Just a bad merge. Sorry for the many balls in the air.